### PR TITLE
Add service to OpenAI to Generate an image

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Add documentation for a new service added to the OpenAI integration.

Parent PR: <https://github.com/home-assistant/core/pull/97018>

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/97018
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/openai_conversation.markdown [hunk 1]
```diff
@@ -48,6 +48,75 @@ Top P:
...
+
+Allows you to ask OpenAI to generate an image based on a prompt. This service
+populates [Response Data](/docs/scripts/service-calls#use-templates-to-handle-response-data)
+with the requested image.
+
+| Service data attribute | Optional | Description                                            | Example          |
+| ---------------------- | -------- | ------------------------------------------------------ | ---------------- |
+| `config_entry`         | no       | Integration entry ID to use.                           |                  |
+| `prompt`               | no       | The text to turn into an image.                        | Picture of a dog |
+| `size`                 | yes      | Size of the returned image in pixels, defaults to 512. | 512              |
```
Comment (c0ffeeca7) [inline source/_integrations/openai_conversation.markdown]:
```suggestion
| `size`                 | yes      | Size of the returned (square) image in pixels, defaults to 512. | 512              |
```
I'm assuming it's the width. Or are they square by default?

**Review by c0ffeeca7 — COMMENTED** [review]
Just one question. Love this service :heart:

File: source/_integrations/openai_conversation.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/openai_conversation.markdown]: Size, it is square :)

Comment (c0ffeeca7) [reply source/_integrations/openai_conversation.markdown]: ok, let me change that.

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**